File: Authorization.Auth/AuthContext.cs
using System.Collections.Generic;
using Authorization.Data.Entities;

namespace Authorization.Auth
{
    public class AuthContext
    {
        public Org Org { get; set; }
        public User User { get; set; }
        public List<Brief> Briefs { get; set; }
        public List<Role> Roles { get; set; }
    }
}

File: Authorization.Web/ClientApp/src/app/models/auth-context.ts
import { Brief } from './api/brief';
import { Org } from './api/org';
import { Role } from './api/role';
import { User } from './api/user';

export interface AuthContext {
  org: Org;
  user: User;
  briefs: Brief[];
  roles: Role[];
}
